logr.go

```diff
@@ -420,6 +444,29 @@ func (logr *Logr) start() {
 	close(logr.done)
 }
 
+// updateMetrics updates the metrics for any polled values every `MetricsUpdateFreqSecs` seconds until
+// logr is closed.
+func (logr *Logr) updateMetrics() {
```

Comment:
Maybe renaming this method, and the target one would be better. I was struggling a bit because I understood that `updateMetrics` was an action instead of a "service" that updates the metrics based on a timer.

What do you think of calling it `runMetricsUpdater` or `startMetricsUpdater`?

Reply:
Renamed to `startMetricsUpdater` via https://github.com/mattermost/logr/pull/2/commits/da17395268f6ff1c9d105225a30305b02271cae9